templates/default/070-components/UI-framework/Table/_ui-component_table.scss

```diff
@@ -334,5 +355,96 @@ td.c-table-data__cell--highlighted {
 }
 // hover on larger screens only
 .c-table-data__row:hover td.c-table-data__cell--highlighted {
-    background-color: darken($il-main-darker-bg, 7%);
-}
\ No newline at end of file
+    @include on-screen-size(large) {
+        background-color: darken($il-main-darker-bg, 7%);
+    }
+}
+
+//
+// Data Table on Small Screens
+//
+
+// Fake Header Title repeated inside the cell, invisible by default
+.c-table-data__cell__col-title {
+    display: none;
+}
+
+.c-table-data {
+    @include on-screen-size(small) {
+        //
+        // Basic Table Design
+        //
+
+        .c-table-data__table,
+        tbody,
+        .c-table-data__cell {
+            width: 100%;
+            display: block;
+            border: none;
+        }
+        tr.c-table-data__row {
+            // using flex so we can pull row action to the top with flex order
+            display: flex;
+            flex-wrap: wrap;
+        }
+
+        // border and padding before/after every entry
+        tr.c-table-data__row {
+            border-top: 1px solid $il-main-border-color;
+            padding: $il-margin-xxxlarge-vertical 0 $il-margin-xxxlarge-vertical;
+        }
+        .c-table-data__table {
+            border-bottom: 1px solid $il-main-border-color;
+        }
+
+        //
+        // Table Header
+        //
+
+        thead {
+            display: none;
+        }
+        // Fake Header Title in Cells
+        .c-table-data__cell__col-title {
+            display: inline;
+            font-weight: bold;
+        }
+
+        //
+        // Column Specific Styling (some resets to get a card look)
+        //
+
+        td.c-table-data__cell--email,
+        td.c-table-data__cell--number {
+            text-align: inherit;
+        }
+        
+        // Highlighted Columns
+        td.c-table-data__cell--highlighted {
+            font-size: $il-font-size-xxlarge;
```

Comment:
@catenglaender There was a dangling reference to `btstrp3.$font-size-h2` which I replaced like this here. Hope that's fine.